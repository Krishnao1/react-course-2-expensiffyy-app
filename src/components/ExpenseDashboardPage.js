import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesSummary from "./ExpensesSummary";
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });


const ExpenseDashboardPage = () => (
  <div>
    <ExpensesSummary/>
    <ExpenseListFilters />
    <ExpenseList />
  </div>
);

export default ExpenseDashboardPage;
