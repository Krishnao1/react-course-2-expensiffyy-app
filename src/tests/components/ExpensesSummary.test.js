import React from 'react'
import {ExpensesSummary} from '../../components/ExpensesSummary'
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallow} from 'enzyme';
configure({ adapter: new Adapter() });


test('should correct render expenses summary ',()=>{
   const wrapper =shallow(<ExpensesSummary expenseCounter={1} expensesTotal={235}/>)
   expect(wrapper).toMatchSnapshot();
})


test('should correct render expenses summary ',()=>{
    const wrapper =shallow(<ExpensesSummary expenseCounter={23} expensesTotal={23566444}/>)
    expect(wrapper).toMatchSnapshot();
})