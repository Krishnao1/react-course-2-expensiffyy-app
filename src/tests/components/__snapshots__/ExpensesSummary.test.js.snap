// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correct render expenses summary  1`] = `ShallowWrapper {}`;

exports[`should correct render expenses summary  2`] = `ShallowWrapper {}`;
